examples/python/example_with_zone.py

```diff
@@ -141,14 +143,14 @@ def write_data(data, connection):
     metadata.create_all(engine)
 
     if args.print:
-        print_data(args.zone_id, connection)
+        print_data(args.zone_id, connection, driver)
```

Comment:
what about place `print('DB driver: {}'.format(driver))` before `if args.print else ` statement and don't pass it to 2 functions with extra parameter?

Reply:
Sure, thanks :-)